py3status/modules/backlight.py

```diff
@@ -36,12 +36,14 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     color = '#FFFFFF'
+    device = None
     device_path = None
```

Comment:
hi I am not exactly sure how to setup the post_config_hook. Here is a small sample app I tried:

```python
class Py3status:
    def post_config_hook(self):
        return 'post config'

    def backlight(self,  i3s_output_list, i3s_config):
        return 'response'

if __name__ == "__main__":
    """
    Run module in test mode.
    """
    from py3status.module_test import module_test
    module_test(Py3status)
```

but this just executes the methods (in alphabetical order). I get the following output:
```
post config
response
...
```

is there something I am missing here?